app/src/main/java/swati4star/createpdf/util/PDFUtils.java

```diff
@@ -589,6 +589,8 @@ public boolean reorderRemovePDF(String inputPath, String output, String pages) {
                         copy.addPage(copy.getImportedPage(reader, startPage));
                         document.close();
                         outputPaths.add(fileName);
+                        new DatabaseHelper(mContext).insertRecord(fileName,
```

Comment:
Rather than duplicating if we put in the ` if (reader.getNumberOfPages() > 1)` 
will not that work ?

Reply:
Yes it will work